pkg/services/sqlstore/migrations/kv_store_mig.go

```diff
@@ -25,3 +25,10 @@ func addKVStoreMigrations(mg *Migrator) {
 
 	mg.AddMigration("add index kv_store.org_id-namespace-key", NewAddIndexMigration(kvStoreV1, kvStoreV1.Indices[0]))
 }
+
+// addKVStoreValueTypeLongTextMigration adds a migration to change the column type of kv_store.value to LONGTEXT for
+// MySQL to be more inline size-wise with PSQL (TEXT) and SQLite.
+func addKVStoreValueTypeLongTextMigration(mg *Migrator) {
+	mg.AddMigration("alter kv_store.value to longtext", NewRawSQLMigration("").
```

Comment:
I wonder if we want to mention MYSQL in migration name

Reply:
Oh boy, I already felt bad with how long it currently is :sweat_smile: but yes I'll add it.